holmscan/scan.py

```diff
@@ -15,6 +13,13 @@ class Scan(HolmscanModule):
     def __init__(self, controller):
         super(Scan, self).__init__(controller)
 
+    def get_net_scan(self, uuid):
+        url = "{0}/net-scans/{1}".format(
+            self.controller.conf.get("HOLMSEC_ENDPOINT"), uuid
```

Comment:
Is the variabel `HOLMSEC_ENDPOINT` something really unique and generic for anyone who uses this? it sounds like we are branding a variabel to a specific customer.

Reply:
HOLMSEC in this case refers to Holm Security. This tool accesses the Holm Security API so the branding is towards the upstream product. We could change this to `HOLMSCAN_ENDPOINT` if that is preferable.